### PR TITLE
BaSP-TG-35: Timesheet Unit Test - getAll, getById & delete

### PR DESCRIPTION
**Pull request by matiasraviola**
[delete] : tested when the id, route are wrong or when you forgot pass the id and when the id is valid
[getAll]: tested when you try to get all the list, filter by each field and when the route is wrong
[getById]: tested when you try to get a valid and/or invalid ID.

### CODE REVIEW

File: src/tests/timesheets.test.js [hunk 1]
```diff
@@ -1 +1,84 @@
...
+const timesheetDate = '2/3/2022';
+const timesheetTask = '63546010fc13ae3a75000197';
+const timesheetEmployee = '6354438cfc13ae204b000064';
+const timesheetProject = '6354438cfc13ae204b000065';
+const timesheetHours = 98;
+
+describe('DELETE /timesheet/:id', () => {
+  test('should delete an employee', async () => {
+    const response = await request(app).delete(`/api/timesheets/${timesheetId}`).send();
+    expect(response.status).toBe(204);
```
Comment (PilarFernandezC) [inline src/tests/timesheets.test.js:21]: you could also try testing the messages like this: `expect(response.body.message).toBe......`

Comment (PilarFernandezC) [inline src/tests/timesheets.test.js:21]: try testing the messages like this: `expect(response.body.message).toBe....... ` See in which tests you could do it.

**Review by PilarFernandezC — CHANGES_REQUESTED** [review]

File: src/tests/timesheets.test.js (hunk 1 above)
Comment (matiasraviola) [reply src/tests/timesheets.test.js:21]: done

**Review by PilarFernandezC — APPROVED** [review]

**Review by paurinaldi — DISMISSED** [review]
dismissed

**Pull request merged**